.github/workflows/recipe_test.yaml

```diff
@@ -43,26 +43,14 @@ jobs:
         with:
           role-to-assume: arn:aws:iam::308535385114:role/gha_workflow_torchtune_pytorch-multimodal
           aws-region: us-east-1
-      - name: Download files from S3
-        uses: nick-fields/retry@v2
-        with:
-          max_attempts: 3
-          retry_on: error
-          timeout_seconds: 14400
-          command: |
-            python3 -m pip install awscli==1.32.6
-            mkdir -p /tmp/test-artifacts
-            aws s3 cp s3://pytorch-multimodal/llama2-7b/tokenizer.model /tmp/test-artifacts
-            aws s3 cp s3://pytorch-multimodal/small-ckpt-01242024 /tmp/test-artifacts
-            aws s3 cp s3://pytorch-multimodal/small-ckpt-hf-03082024.pt /tmp/test-artifacts
-            aws s3 cp s3://pytorch-multimodal/small-ckpt-tune-03082024.pt /tmp/test-artifacts
-            aws s3 cp s3://pytorch-multimodal/small-ckpt-meta-03082024.pt /tmp/test-artifacts
+      - name: Install S3 CLI
+        run: |
+          python3 -m pip install awscli==1.32.6
```

Comment:
Curious if there's a reason to pin to this version? If so, maybe a comment to make sure no one inadvertently changes this?

Reply:
Honestly I have no idea, I am just following along with what we had previously here. Can take a look though